packages/tracing-internal/src/node/integrations/express.ts

```diff
@@ -366,6 +368,76 @@ type LayerRoutePathInfo = {
   numExtraSegments: number;
 };
 
+/**
+ * Recreate layer.route.path from layer.regexp and layer.keys.
+ * Works until express.js used package path-to-regexp@0.1.7
```

Comment:
Btw, if this no-ops for a certain version range that's probably fine, I just want to avoid breaking users if this function would throw an error for some versions.

Reply:
Ok, so as long as all _current_ Express 4 versions use a version of this library that works, I'm fine with that.